apheleia.el

```diff
@@ -545,28 +624,32 @@ If INTERACTIVE is non-nil, then prompt the user for which
 formatter to run if none is configured, instead of returning nil.
 If INTERACTIVE is the special symbol `prompt', then prompt
 even if a formatter is configured."
-  (when-let ((formatter
+  (when-let ((formatters
               (or (and (not (eq interactive 'prompt))
-                       (or apheleia-formatter
+                       (or (apheleia--ensure-list apheleia-formatter)
                            (cl-dolist (entry apheleia-mode-alist)
                              (when (or (and (symbolp (car entry))
                                             (derived-mode-p (car entry)))
                                        (and (stringp (car entry))
                                             buffer-file-name
                                             (string-match-p
                                              (car entry) buffer-file-name)))
-                               (cl-return (cdr entry))))))
+                               (cl-return
+                                (apheleia--ensure-list (cdr entry)))))))
```

Comment:
It feels like we could hoist the call to `apheleia--ensure-list` outside of the enclosing `or`, and thereby avoid duplicating the call.